indico/MaKaC/webinterface/tpls/ConfDetailsFull.tpl

```diff
@@ -122,8 +120,9 @@
         </div>
     </div>
   % endif
-  </div>
 
+  ${ template_hook('conference-home-info', event=conf) }
```

Comment:
indent? (even though this template is already a mess so it probably doesn't really matter)